packages/govern-lib/src/GovernLib.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  Address,
+  DaoData,
+  Networkish,
+  RegistryEntryData,
+  GovernQueueData,
+} from './types'
+import { ErrorInvalidNetwork, ErrorUnexpectedResult } from './errors'
+import NetworkContext from './utils/NetworkContext'
+import GraphqlClient from './GraphqlClient'
+import {
+  QUERY_DAO,
+  QUERY_DAOS,
+  QUERY_REGISTRY_ENTRY,
+  QUERY_REGISTRY_ENTRIES,
+  QUERY_QUEUE,
+  QUERY_QUEUES,
+  QUERY_QUEUES_BY_DAO,
+} from './thegraph-queries'
+
+export type GovernLibConfig = {
+  network: Networkish
+  subgraphUrl?: string
+  verbose?: boolean
+}
+
+class GovernLib {
+  #gql: GraphqlClient
+  readonly config: GovernLibConfig
+  readonly networkContext: NetworkContext
+
+  constructor(config: GovernLibConfig) {
```

Comment:
Would be great to have funcDocs for all methods. It increases the readability especially if always a short helpful comment is existing.

Reply:
True, will add